docs/skill_development/skill-structure/overview.md

```diff
@@ -0,0 +1,204 @@
+
+We will look at each of these in turn.
+
+## `locale` directory
+
+The `locale` directory contain subdirectories for each spoken language the skill supports. The subdirectories are named using the [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag) for the language. For example, Brazilian Portugues is 'pt-br', German is 'de-de', and Australian English is 'en-au'.
+
+By default, your new Skill should contain one subdirectory for United States English - 'en-us'. If more languages were supported, then there would be additional language directories.
+
+Note that everything under "language" can be in any structure but is generally organized by resource type: dialog, vocab, regex, ui.
+
+```bash
+$ ls -l locale
+total 4
+drwxr-xr-x 2 kris kris 4096 Oct  8 22:21 en-us
+```
+
+### Dialog Directory
+
+There will be one file in the language subdirectory (ie. `en-us`) for each type of dialog the Skill will use. Currently this will contain all of the phrases you input when creating the Skill.
+
+```bash
+$ ls -l locale/en-us/dialog
+total 4
+-rw-r--r-- 1 kris kris 10 Oct  8 22:21 first.dialog
+```
+
+When instructed to use a particular dialog, Neon will choose one of these lines at random. This is closer to natural speech. That is, many similar phrases mean the same thing.
+
+For example, how do you say 'goodbye' to someone?
+
+- Bye for now
+- See you round
+- Catch you later
+- Goodbye
+- See ya!
+
+#### Vocab Directory
+
+Each Skill defines one or more Intents. Intents are defined in the `vocab` directory. The `vocab` directory is organized by language, just like the `dialog` directory.
+
+We will learn about Intents in more detail shortly. For now, we can see that within the `vocab` directory you may find multiple types of files:
+
+- `.intent` files used for defining Padatious/Padacioso Intents
+- `.voc` files define keywords primarily used in Adapt Intents
+- `.entity` files define a named entity also used in Adapt Intents
+
+In our current example we might see something like:
+
+```bash
+$ ls -l locale/en-us/vocab
+total 4
+-rw-r--r-- 1 kris kris 23 Oct  8 22:21 first.intent
+```
+
+This `.intent` file will contain all of the sample utterances we provided when creating the Skill.
+
+### Regex Directory
+
+Contains all the files for any regex intents the skill uses (optional). For more information, [see the regex documentation](../user-interaction/intents/adapt-intents.md#regular-expression-rx-files).
+
+### UI Directory
+
+Contains all the files for the GUI of the skill (optional). For more information, [see the GUI documentation](../displaying-information/mycroft-gui.md).
+
+## \_\_init\_\_.py
+
+The `__init__.py` file is where most of the Skill is defined using Python code. We will learn more about the contents of this file in the next section.
+
+Let's take a look:
+
+### Importing libraries
+
+```python
+from ovos_utils.intents import IntentBuilder
+from ovos_workshop.decorators import intent_handler
+from Neon import NeonSkill
```

Comment:
Missed before but this import is wrong

`from neon_utils.skills.neon_skill import NeonSkill`